internal/service/zoom_webhook_handlers.go

```
@@ -170,20 +195,51 @@ func (s *MeetingsService) HandleZoomSummaryCompleted(ctx context.Context, msg do
 
 // handleMeetingStartedEvent processes meeting.started events
 func (s *MeetingsService) handleMeetingStartedEvent(ctx context.Context, event models.ZoomWebhookEventMessage) error {
+	slog.InfoContext(ctx, "processing meeting started event")
+
 	// Convert to typed payload
 	payload, err := event.ToMeetingStartedPayload()
 	if err != nil {
-		slog.ErrorContext(ctx, "failed to convert to typed meeting started payload", "error", err)
-		return fmt.Errorf("failed to parse meeting started payload: %w", err)
+		slog.ErrorContext(ctx, "failed to parse meeting started payload", logging.ErrKey, err)
+		return fmt.Errorf("invalid meeting.started payload: %w", err)
 	}
 
-	slog.InfoContext(ctx, "processing meeting started event",
-		"zoom_meeting_uuid", payload.Object.UUID,
-		"zoom_meeting_id", payload.Object.ID,
-		"topic", payload.Object.Topic,
-		"start_time", payload.Object.StartTime,
+	meetingObj := payload.Object
+
+	slog.InfoContext(ctx, "meeting started",
+		"zoom_meeting_id", meetingObj.ID,
+		"zoom_meeting_uuid", meetingObj.UUID,
+		"topic", meetingObj.Topic,
+		"start_time", meetingObj.StartTime,
+		"host_id", meetingObj.HostID,
 	)
 
+	// Find the meeting by Zoom meeting ID
+	meeting, err := s.MeetingRepository.GetByZoomMeetingID(ctx, meetingObj.ID)
+	if err != nil {
+		slog.ErrorContext(ctx, "failed to find meeting by Zoom ID", logging.ErrKey, err)
+		return fmt.Errorf("meeting not found for Zoom ID %s: %w", meetingObj.ID, err)
+	}
+
+	// Create the past meeting record
+	zoomData := ZoomPayloadForPastMeeting{
+		UUID:      meetingObj.UUID,
+		StartTime: meetingObj.StartTime,
+		EndTime:   nil, // meeting.started events don't have end time
+		Timezone:  meetingObj.Timezone,
+	}
+	pastMeeting, err := s.createPastMeetingRecord(ctx, meeting, zoomData)
+	if err != nil {
```

In general, do we need a `logging.PriorityCritical()` for observability/alerting purposes in event handling? To avoid failing silently.

---

Yeah I think it makes a lot of sense, so I'll add some critical logging.